fdbclient/VersionVector.h

```diff
@@ -144,9 +161,365 @@ struct VersionVector {
 	bool operator!=(const VersionVector& vv) const { return maxVersion != vv.maxVersion; }
 	bool operator<(const VersionVector& vv) const { return maxVersion < vv.maxVersion; }
 
+	//
+	// Methods to set/get/check/invalidate cached encoded version vector size.
+	//
+	void setCachedEncodedSize(size_t size) { cachedEncodedSize = size; }
+
+	bool isEncodedSizeCached() const { return cachedEncodedSize > 0; }
+
+	size_t getCachedEncodedSize() const {
+		ASSERT(isEncodedSizeCached());
+		return cachedEncodedSize;
+	}
+
+	void invalidateCachedEncodedSize() { cachedEncodedSize = 0; }
```

Comment:
Can be moved to "private" section as an `inline` function.

Reply:
Ok.

Reply:
Done.